src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraAuthorizationException.scala

```diff
@@ -0,0 +1,16 @@
+
+package org.broadinstitute.dsde.agora.server.business
+
+import org.broadinstitute.dsde.agora.server.dataaccess.acls.AgoraPermissions
+import org.broadinstitute.dsde.agora.server.model.AgoraEntity
+
+case class AgoraAuthorizationException(agoraPermissions: AgoraPermissions, agoraEntity: AgoraEntity) extends Exception {
+  override def getMessage: String = {
+    "Authorization exception attempting to exercise permission %d on entity %s.%s.%d. %s".format(
```

Comment:
Should we prefer string interpolation to format?
s"Authorization exception attempting to exercise permission $agoraPermissions.permissions

It would make this string longer, but you could break it up over multiple lines with +

Up to you which is more readable